source/quick-start.txt

```diff
@@ -81,7 +116,12 @@ After completing this step, you should have a working application that uses
 the {+driver-short+} to connect to your MongoDB cluster, run a query on the
 sample data, and print out the result.
 
+To learn more about connecting to Atlas with the {+driver-short+}, see
+the :atlas:`Atlas driver connection </driver-connection>` guide
+and select **{+language+}** from the *Select your language* dropdown.
```

Comment:
```suggestion
and select **{+language+}** from the :guilabel`Select your language` dropdown.
```

Reply:
I think the language should probably also use the guilabel per the style guide. I'm not sure how useful using a source constant is here since it should match the Atlas UI representation which could change.